vpnc fails with "RTNETLINK answers: No such device"
Patch from: https://bugs.gentoo.org/attachment.cgi?id=245736
In reference to: https://bugs.gentoo.org/show_bug.cgi?id=331447

--- iproute2-2.6.35.old/ip/iproute.c	2010-09-02 16:00:21.805000124 +0200
+++ iproute2-2.6.35/ip/iproute.c	2010-09-02 16:00:40.782000125 +0200
@@ -160,7 +160,7 @@
 	if (r->rtm_family == AF_INET6 && table != RT_TABLE_MAIN)
 		ip6_multiple_tables = 1;
 
-	if (filter.cloned == !(r->rtm_flags&RTM_F_CLONED))
+	if (filter.cloned && !(r->rtm_flags&RTM_F_CLONED))
 		return 0;
 
 	if (r->rtm_family == AF_INET6 && !ip6_multiple_tables) {
